yarn.lock

```diff
@@ -6972,7 +6972,7 @@ cross-fetch@3.0.5:
   dependencies:
     node-fetch "2.6.0"
 
-cross-fetch@^3.0.4:
+cross-fetch@3.0.6, cross-fetch@^3.0.4:
```

Comment:
why not `node-fetch`?

Reply:
We used that one on the first place, but there was a compatibility issue with `apollo-link-http`.

Reply:
oh, what was the problem?

Reply:
In `fetchMangement`, we construct a `httpLink` with `apollo-link-http` for GraphQL queries & mutations, with API url & `fetch`. But as the new scripts launched in GH actions are running in Node env, we can't use the global `fetch`provided by NextJs, so we needed an external one. And sadly, `node-fetch` was not compatible with `apollo-link-http` (more precisely, it seems to be a type issue).

Reply:
https://github.com/apollographql/apollo-link/issues/513#issuecomment-384743835

Reply:
We could add `isomorphic-fetch` too if you prefer Fenn.

Reply:
Well no, let's go with `cross-fetch` since it's just an `if` around `node-fetch` anyway. I'm annoyed at the appolo team since https://github.com/apollographql/apollo-link/issues/513#issuecomment-436364057 
I don't know where the faulty types lies (could be from apollo, from node-fetch or even be the wrong type in both) but the fact that they rely on the fact that `cross-fetch` or `isomorphic-fetch` have a a type that goes better with their code and don't work with the other ones is not a good look in my opinion.

That being said, if `cross-fetch` does the job ¯\_(ツ)_/¯

Reply:
I agree, good thing we are moving away from GraphQL for Account then I guess.